pkg/bootstrap/config.go

```diff
@@ -58,13 +58,8 @@ const (
 
 // Config for creating a bootstrap file.
 type Config struct {
-	Node                string
-	Proxy               *meshAPI.ProxyConfig
-	PlatEnv             platform.Environment
+	*model.Node
 	PilotSubjectAltName []string
```

Comment:
These flags need to map to bootstrap metadata. Since that is a user-visible change, it's left out.